MediaBrowser.Providers/Tmdb/Movies/TmdbSearch.cs

```diff
@@ -18,8 +19,22 @@ namespace MediaBrowser.Providers.Tmdb.Movies
 {
     public class TmdbSearch
     {
-        private static readonly CultureInfo EnUs = new CultureInfo("en-US");
-        private const string Search3 = TmdbUtils.BaseTmdbApiUrl + @"3/search/{3}?api_key={1}&query={0}&language={2}";
+        private static readonly CultureInfo _usCulture = new CultureInfo("en-US");
+
+        private static readonly Regex _cleanEnclosed = new Regex(@"\p{Ps}.*\p{Pe}", RegexOptions.Compiled);
+        private static readonly Regex _cleanNonWord = new Regex(@"[\W_]+", RegexOptions.Compiled);
+        private static readonly Regex _cleanStopWords = new Regex(@"\b( # Start at word boundary
+            19[0-9]{2}|20[0-9]{2}| # 1900-2099
+            S[0-9]{2}| # Season
+            E[0-9]{2}| # Episode
```

Comment:
any reason to assume season and episode are always 2-digit? why not just `[0-9]+`?

Reply:
We currently have an issue with episodes that have 3 digits fyi

Reply:
i will fix that

Reply:
bump :) just in case